xtc/src/ledger.rs

```diff
@@ -284,37 +396,144 @@ mod tests {
         Principal::from_text("ai7t5-aibaq-aaaaa-aaaaa-c").unwrap()
     }
 
-    fn john() -> Principal {
+    fn charlie() -> Principal {
```

Comment:
Ha, I didn't realise I had briefly made it into the test suite.

Reply:
:D fixed the problem

Reply:
@kanej You're also in `ic_kit` mock principals, so you're gonna be in a lot of people's test suites.